pkg/converter/converter.go

```diff
@@ -122,7 +122,9 @@ func (cvt *Converter) Convert(ctx context.Context, source, target, cacheRef stri
 
 	logger.Infof("converting image %s", source)
 	start = time.Now()
-	cvt.provider.SetCacheRef(cacheRef)
+	if err := cvt.provider.ClearCache(cacheRef); err != nil {
```

Comment:
How can we handle concurrency here?

Reply:
Maybe cache in content  is not just a map, needs to distinguish between different source images, such as a map which key is source image reference, like `map[string] map[string]ocispec.Descriptor`.Then we can clear the specified image cache ?

Reply:
> How can we handle concurrency here?

Maybe cache in content  is not just a map, needs to distinguish between different source images, such as a map which key is source image reference, like `map[string] map[string]ocispec.Descriptor`.Then we can clear the specified image cache ?